test/parallel/test-crypto-dh.js

```diff
@@ -248,7 +248,7 @@ if (availableCurves.has('prime256v1') && availableCurves.has('secp256k1')) {
     () => ecdh2.computeSecret(key3, 'latin1', 'buffer'),
     {
       code: 'ERR_CRYPTO_ECDH_INVALID_PUBLIC_KEY',
-      type: Error,
+      type: TypeError,
```

Comment:
Seems like I made a mistake here. I think this should stay as `Error`.

Reply:
You are right. I fixed it.